trunner/runners/common.py

```diff
@@ -343,6 +370,97 @@ def run(self, test):
             self.serial.close()
 
 
+class ARMV7M7Runner(DeviceRunner, ABC):
+    """This class provides interface to run tests on targets with armv7m7 architecture"""
+
+    def __init__(
+        self,
+        port,
+    ):
+        # default values, redefined by specified target runners
+        self.is_cut_power_used = False
+        self.flash_memory = 0
+        super().__init__(port)
+
+    def _restart_by_jtag(self):
+        self.reset_gpio.low()
```

Comment:
we're using instance attribute not defined in the same class, only in the childs.

Please define it in this class (to be overriden in subclasses if needed).